adflow/pyADflow.py

```diff
@@ -1040,14 +1041,15 @@ def __call__(self, aeroProblem, **kwargs):
                 )
 
         if self.adflow.killsignals.fatalfail:
-            print("Fatal failure during mesh warp! Bad mesh is " "written in output directory as failed_mesh.cgns")
-            fileName = os.path.join(self.getOption("outputDirectory"), "failed_mesh.cgns")
-            self.writeMeshFile(fileName)
+            fileName = f"failed_mesh_{self.nMeshFail:02}.cgns"
```

Comment:
I think it would be useful to number the mesh failures using the normal call counter. That way you can easily where in the iteration history the failed mesh goes.

Reply:
Oh yeah this is actually a good point. Would you mind opening a subsequent PR to patch this?